docs/01-get-started/your-first-route.mdx

```diff
@@ -6,12 +6,48 @@ sidebar_position: 3
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+# Routes
```

Comment:
**nitpick:** there might be other ways to create routes in FC (and there is in Remix too)
```suggestion
# File-based routing
```